src/Components/Todos/TodoRows.js

```diff
@@ -48,17 +48,17 @@ export const TodoRows = (props) => {
       <span
         className="todo-del-right"
         aria-hidden="true"
-        onClick={() => (props.onTodoClick ? props.onTodoClick(props.todo.id, props.actions[0]) : _ => _)}
+        onClick={() => (props.onTodoClick ? props.onTodoClick(props.todo, props.actions[0]) : _ => _)}
       />
       <span
         className="todo-compl-right"
         aria-hidden="true"
-        onClick={() => props.onTodoClick(props.todo.id, props.actions[1])}
+        onClick={() => props.onTodoClick(props.todo, props.actions[1])}
       />
       <span
         className="todo-proc-right"
         aria-hidden="true"
-        onClick={() => props.onTodoClick(props.todo.id, props.actions[2])}
+        onClick={() => props.onTodoClick(props.todo, props.actions[2])}
       />
     </div >
   );
```

Comment:
```
if (props.todo.done) {..}
if (props.todo.done === false) {...}

return....
```

There is really almost the same code in three places. You should use logical operators in *one* JSX expression, but not three separate JSX expressions